pkg/network/usm/compile_test.go

```diff
@@ -28,8 +29,11 @@ func TestUSMCompile(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func rtcHTTPSupported(t *testing.T) bool {
+func rtcHTTPSupported() bool {
 	currKernelVersion, err := kernel.HostVersion()
-	require.NoError(t, err)
+	if err != nil {
```

Comment:
Not failing here as in other places?

Reply:
that function is used only once, I think I'll just merge it with TestHTTPCompile and let it fail